app/controllers/admin/documentations_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Admin::DocumentationsController < AdminController
   before_action :assign_documentation, only: %i(show edit update destroy)
+  before_action :pluck_cover_path_documentation, only: %i(new edit)
```

Comment:
Good idea! 👍 
To stay coherent with the above before_action, you could rename this method `assign_documentations_cover_path`.